### PR TITLE
Run integration tests

### LINKED ISSUE #20: Run integration tests
It's possible to run docker container based integration test in this test environment.

### PR DESCRIPTION
**Pull request by hristobojilov**
[#20] Run integration tests

Signed-off-by: Hristo Bozhilov <Hristo.Bozhilov@bosch.io>

### CODE REVIEW

File: .github/workflows/validation.yaml [hunk 1]
```diff
@@ -41,5 +41,6 @@ jobs:
           go vet ./...
       - name: Test
         run: |
-          go test ./... -coverprofile coverage.out -covermode count -tags=unit
```
Comment (e-grigorov) [inline .github/workflows/validation.yaml:44]: Why are the unit tests removed?

File: routing/it_discovery_test.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+			}
+			break loop
+
+		case <-ctx.Done():
+			break loop
+		}
+	}
+
+	assert.Contains(t, params, "deviceId")
+	assert.Contains(t, params, "tenantId")
```
Comment (e-grigorov) [inline routing/it_discovery_test.go]: policyId can be checked also.

**Review by e-grigorov — CHANGES_REQUESTED** [review]

File: .github/workflows/validation.yaml (hunk 1 above)
Comment (hristobojilov) [reply .github/workflows/validation.yaml:44]: All tests are run and nothing is removed. Tests can be run indirectly as well.

File: routing/it_discovery_test.go (hunk 2 above)
Comment (e-grigorov) [reply routing/it_discovery_test.go]: TC is not required and has been removed.

**Review by e-grigorov — APPROVED** [review]

**Pull request merged**